### PR TITLE
feat(route): 观察者网输出全文

### LINKED ISSUE #9335: 观察者网获取全文
### 这是一个什么样的功能？

观察者网默认对长文进行折叠,.
实测在网址的`.shtml`前添加`_s`即为全文. 
如[当前默认](https://www.guancha.cn/politics/2022_03_18_630889.shtml)
和[添加后](https://www.guancha.cn/politics/2022_03_18_630889_s.shtml)

如果是风闻, 则是在最后添加param: `&page=0`
如: https://user.guancha.cn/main/content?id=718132&page=0
(原来是: https://user.guancha.cn/main/content?id=718132 )

### 这个功能可以解决什么问题？

获取被折叠的完整内容

### 额外描述

_No response_

### 这不是重复的功能请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues)，以确保这项功能尚未被请求。

### PR DESCRIPTION
**Pull request by nczitzk**
## 该 PR 相关 Issue / Involved issue

Close #9335

## 完整路由地址 / Example for the proposed route(s)

```routes
/guancha
/guancha/headline
/guancha/topic/110/1
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [ ] 新的路由 New Route
  - [ ] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [ ] 文档说明 Documentation
  - [ ] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/guancha/radar.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'guancha.cn': {
+        _name: '观察者网',
+        '': [
+            {
+                title: '用户收藏',
+                docs: 'https://docs.rsshub.app/new-media.html#guan-cha-zhe-wang',
+                source: '/bookmark.php',
+                target: (params, url) => `/pixiv/user/bookmarks/${new URL(url).searchParams.get('id')}`,
```
Comment (TonyRL) [inline lib/v2/guancha/radar.js:9]: Wrong target

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/guancha/index.js [hunk 2]
```diff
@@ -112,9 +138,9 @@ module.exports = async (ctx) => {
                 if (dateMatch === null) {
                     // PubDates of posts in 'fengwen' are in an informal format.
 
-                    item.pubDate = date(content('.time1').text(), 8);
+                    item.pubDate = parseRelativeDate(content('.time1').text());
                 } else {
-                    item.pubDate = new Date(dateMatch[1]).toUTCString();
+                    item.pubDate = parseDate(dateMatch[1]);
```
Comment (TonyRL) [inline lib/v2/guancha/index.js:143]: Need timezone adjustment

File: lib/v2/guancha/radar.js (hunk 1 above)
Comment (nczitzk) [reply lib/v2/guancha/radar.js:9]: > Wrong target

WIP

File: lib/v2/guancha/radar.js [hunk 3]
```diff
@@ -3,10 +3,46 @@ module.exports = {
         _name: '观察者网',
         '': [
```
Comment (TonyRL) [inline lib/v2/guancha/radar.js:4]:
```suggestion
        '.': [
```

**Pull request merged**